src/backend/db/types.js

```diff
@@ -0,0 +1,37 @@
+import { v4 as uuid } from "uuid";
+
+/**
```

Comment:
Here too the comments can be removed

Reply:
These comments came with backend and help as doc.